xml/http_wwwauth.xml

```diff
@@ -167,18 +170,35 @@
     <example>Digest realm="RUIJIE-CPE"</example>
     <param pos="0" name="hw.vendor" value="Ruijie"/>
   </fingerprint>
-  <fingerprint pattern="^(?:Basic|Digest) realm=.SpeedTouch \(([0-9A-F]{2}-[0-9A-F]{2}-[0-9A-F]{2}-[0-9A-F]{2}-[0-9A-F]{2}-[0-9A-F]{2})\).$">
+  <fingerprint pattern="^Basic realm=&quot;SpeedTouch&quot;$">
+    <description>Thomson SpeedTouch xDSL router - short variant</description>
+    <example>Basic realm="SpeedTouch"</example>
+    <param pos="0" name="service.vendor" value="Thomson"/>
+    <param pos="0" name="service.product" value="SpeedTouch"/>
+    <param pos="0" name="service.family" value="SpeedTouch"/>
+    <param pos="0" name="os.vendor" value="Thomson"/>
+    <param pos="0" name="os.device" value="Broadband router"/>
+    <param pos="0" name="os.family" value="SpeedTouch"/>
+    <param pos="0" name="hw.vendor" value="Thomson"/>
+    <param pos="0" name="hw.family" value="SpeedTouch"/>
+    <param pos="0" name="hw.device" value="Broadband router"/>
+  </fingerprint>
+  <fingerprint pattern="^(?:Basic|Digest) realm=&quot;SpeedTouch \(([0-9A-F]{2}-[0-9A-F]{2}-[0-9A-F]{2}-[0-9A-F]{2}-[0-9A-F]{2}-[0-9A-F]{2})\)&quot;$">
     <description>Thomson SpeedTouch xDSL router</description>
+    <example host.mac="00-90-D0-F9-91-52">Basic realm="SpeedTouch (00-90-D0-F9-91-52)"</example>
     <param pos="0" name="service.vendor" value="Thomson"/>
     <param pos="0" name="service.product" value="SpeedTouch"/>
     <param pos="0" name="service.family" value="SpeedTouch"/>
     <param pos="0" name="os.vendor" value="Thomson"/>
     <param pos="0" name="os.device" value="Broadband router"/>
     <param pos="0" name="os.family" value="SpeedTouch"/>
     <param pos="0" name="os.product" value="SpeedTouch"/>
+    <param pos="0" name="hw.vendor" value="Thomson"/>
+    <param pos="0" name="hw.family" value="SpeedTouch"/>
+    <param pos="0" name="hw.device" value="Broadband router"/>
     <param pos="1" name="host.mac"/>
   </fingerprint>
-  <fingerprint pattern="^(?:Basic|Digest) realm=.SpeedTouch., nonce=.[0-9A-Z]+:([0-9A-F]{2}-[0-9A-F]{2}-[0-9A-F]{2}-[0-9A-F]{2}-[0-9A-F]{2}-[0-9A-F]{2}):\d+:\d+., qop=.auth.$">
+  <fingerprint pattern="^(?:Basic|Digest) realm=&quot;SpeedTouch., nonce=.[0-9A-Z]+:([0-9A-F]{2}-[0-9A-F]{2}-[0-9A-F]{2}-[0-9A-F]{2}-[0-9A-F]{2}-[0-9A-F]{2}):\d+:\d+., qop=.auth&quot;$">
```

Comment:
This one and the one above it are identical with the exception of the `qop` part which isn't reflected in the resulting fingerprint anyway.  Combine into one.

Reply:
Yup, looks like one was derived from Digest and the other basic.